docs/index.rst

```diff
@@ -1,159 +1,325 @@
 Testimony
 =========
 
-An approach to document test cases in the Python source code
-------------------------------------------------------------
+.. contents:: Topics
 
-Are you tired of managing your test cases in a test case management tool and
-your test code in a Python automation framework?  Testimony can help you to use
-your Python automation framework as a test case repository tool.
+.. _what_is_testimony:
```

Comment:
It works great locally, it seems to be an issue with how readthedocs processes these specific attributes.